### PR TITLE
add description of covidcast data

### PR DESCRIPTION
**Pull request by pedritom-amzn**

### CODE REVIEW

File: docs/api/covidcast.md [hunk 1]
```diff
@@ -24,13 +33,20 @@ See [this documentation](README.md) for details on specifying epiweeks, dates, a
...
 | `geo_type` | spatial resolution of the signal (e.g., `county`, `hrr`, `msa`, `dma`, `state`) | string |
 | `time_values` | time unit (e.g., date) over which underlying events happened | `list` of time values (e.g., 20200401) |
 | `geo_value` | unique code for each location, depending on `geo_type` (county -> FIPS 6-4 code, HRR -> HRR number, MSA -> CBSA code, DMA -> DMA code, state -> two-letter [state](../../labels/states.txt) code), or `*` for all | string |
 
+Notes:
+* `fb_survey` `signal` values include `cli`, `ili`, and `wili`.
+* `google_survey` `signal` values include `cli`.
+* `ght` `signal` values include `rawsearch` and `smoothedsearch`.
+* `quidel` `signal` values include `raw_negativeprop` and `smooth_negativeprop`.
+* `doctor_visits` `signal` values include `cli`.
```
[the remark this answers is not in the document]
Comment (undefx) [reply docs/api/covidcast.md:48]: to be clear, just requesting dashes for **data source name**. signal name can continue using underscores.

File: docs/api/covidcast.md [hunk 2]
```diff
@@ -10,7 +10,16 @@ General topics not specific to any particular data source are discussed in the
 
 ## Delphi's COVID-19 Surveillance Streams Data
 
-... <!-- TODO -->
+Delphi's COVID-19 Surveillance Streams data includes the following data sources:
+* `fb_survey`: Data signal based on CMU-run symptom surveys, advertised through Facebook.  These surveys are voluntary, and no individual survey responses are shared back to Facebook. Using this survey data, we estimate the percentage of people in a given location, on a given day that have CLI (covid-like illness = fever, along with cough, or shortness of breath, or difficulty breathing), and separately, that have ILI (influenza-like illness = fever, along with cough or sore throat).
```
[the remark this answers is not in the document]
Comment (pedritom-amzn) [reply docs/api/covidcast.md]: updated

File: docs/api/covidcast.md [hunk 3]
```diff
@@ -24,13 +33,20 @@ See [this documentation](README.md) for details on specifying epiweeks, dates, a
 
 | Parameter | Description | Type |
 | --- | --- | --- |
-| `data_source` | name of upstream data souce | string |
-| `signal` | name of signal derived from upstream data | string |
+| `data_source` | name of upstream data source (e.g., `fb_survey`, `google_survey`, `ght`, `quidel`, `doctor_visits`) | string |
```
[the remark this answers is not in the document]
Comment (pedritom-amzn) [reply docs/api/covidcast.md]: updated

File: docs/api/covidcast.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (pedritom-amzn) [reply docs/api/covidcast.md]: updated

**Comment by pedritom-amzn** [comment]
data source names with _ updated to use -.

**Pull request merged**